includes/definitions/discovery/innovaphone.yaml

```diff
@@ -0,0 +1,64 @@
+mib: INNO-MIB
+modules:
+    sensors:
+        state:
+            data:
+                -
+                    oid: l2Table
+                    value: l2State
+                    num_oid: '.1.3.6.1.4.1.6666.1.1.1.2.{{ $index }}'
+                    descr: "LAPD {{ $l2Label }}"
+                    state_name: L2State
+                    states:
+                        - { descr: down, graph: 0, value: 1, generic: 2 }
+                        - { descr: up,   graph: 0, value: 2, generic: 0 }
+                -
+                    oid: l2Table
+                    value: l2Mode
+                    num_oid: '.1.3.6.1.4.1.6666.1.1.1.3.{{ $index }}'
+                    descr: 'Interface Mode'
+                    state_name: L2Mode
+                    states:
+                        - { descr: TE, graph: 0, value: 1, generic: 0 }
+                        - { descr: NT, graph: 0, value: 2, generic: 0 }
+
+                -
+                    oid: l1BriTable
+                    value: l1BriState
+                    num_oid: '.1.3.6.1.4.1.6666.1.1.1.3'
```

Comment:
This num OID is the same as the previous one (so one of the 2 is wrong) and does not include index.